docs/topics/information_sources.md

```diff
@@ -89,11 +89,9 @@ This ambiguity makes it impossible to cleanly map the [*Technical Impact*](../re
 
 As mentioned in the discussion of [*Exploitation*](../reference/decision_points/exploitation.md), [CWE](https://cwe.mitre.org/) could be used to inform one of the conditions that satisfy [*proof of concept*](../reference/decision_points/exploitation.md).
 For some classes of vulnerabilities, the proof of concept is well known because the method of exploitation is already part of open-source tools.
-For example, on-path attacker scenarios for intercepting TLS certificates.
+An example of this is on-path attacker scenarios for intercepting TLS certificates.
 These scenarios are a cluster of related vulnerabilities.
-Since CWE classifies clusters of related vulnerabilities, the community could likely curate a list of CWE-IDs for which this condition of well known exploit technique is satisfied.
-Once that list were curated, it could be used to automatically populate a CVE-ID as [*proof of concept*](../reference/decision_points/exploitation.md) if the CWE-ID of which it is an instance is on the list.
-Such a check could not be exhaustive, since there are other conditions that satisfy [*proof of concept*](../reference/decision_points/exploitation.md).
+We provide a non-exhaustive [list of CWE-IDs with known proofs of concept](../reference/decision_points/exploitation/#cwe-ids-for-poc). This is list is non-exhaustive becuase there are other conditions that satisfy [*proof of concept*](../reference/decision_points/exploitation.md).
```

Comment:
link checker is choking on the first url in this line. I think it needs to end in `exploitation.md#cwe-ids-for-poc` instead